src/views/Orders.vue

```diff
@@ -27,24 +27,25 @@
               </ion-label>
               <div class="metadata">
                 <ion-badge v-if="order.orderDate" color="dark">{{ moment.utc(order.orderDate).fromNow() }}</ion-badge>
-                <ion-badge v-if="order.statusId !== 'ORDER_APPROVED'" color="danger">{{ $t('pending approval') }}</ion-badge>
+                <ion-badge v-if="order.orderStatusId !== 'ORDER_APPROVED'" color="danger">{{ $t('pending approval') }}</ion-badge>
```

Comment:
This should be extracted from statusId only

Reply:
Sir, we are not getting status id in response, that's why I used this orderStatusId.